tfhe/src/core_crypto/algorithms/lwe_multi_bit_programmable_bootstrapping.rs

```diff
@@ -1603,42 +1540,31 @@ pub fn std_multi_bit_deterministic_blind_rotate_assign<Scalar, InputCont, Output
         "Got thread_count == 0, this is not supported"
     );
 
-    let (lwe_mask, lwe_body) = input.get_mask_and_body();
+    assert!(accumulator
+        .ciphertext_modulus()
+        .is_compatible_with_native_modulus());
```

Comment:
an error message please, if we are missing that in other multi bit primitives feel free to update

Reply:
Added